src/tokens-studio/spectrum2-non-colors/spectrum2/layout/desktop.json

```diff
@@ -747,6 +769,237 @@
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "corner-radius-500",
+        "constant-token-duplicate": false
+      }
+    }
+  },
+  "corner-radius-600": {
+    "value": "9px",
+    "type": "borderRadius",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "corner-radius-600",
+        "constant-token-duplicate": false
+      }
+    }
+  },
+  "corner-radius-700": {
+    "value": "10px",
+    "type": "borderRadius",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "corner-radius-700",
+        "constant-token-duplicate": false
+      }
+    }
+  },
+  "corner-radius-800": {
+    "value": "16px",
+    "type": "borderRadius",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "corner-radius-800",
+        "constant-token-duplicate": false
+      }
+    }
+  },
+  "corner-radius-none": {
+    "value": "{corner-radius-0}",
+    "type": "borderRadius",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "corner-radius-none",
+        "constant-token-duplicate": false
+      }
+    }
+  },
+  "corner-radius-small-default": {
+    "value": "{corner-radius-100}",
+    "type": "borderRadius",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "corner-radius-small-default",
+        "constant-token-duplicate": false
+      }
+    }
+  },
+  "corner-radius-medium-default": {
+    "value": "{corner-radius-500}",
+    "type": "borderRadius",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "corner-radius-medium-default",
+        "constant-token-duplicate": false
+      }
+    }
+  },
+  "corner-radius-large-default": {
+    "value": "{corner-radius-700}",
+    "type": "borderRadius",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "corner-radius-large-default",
+        "constant-token-duplicate": false
+      }
+    }
+  },
+  "corner-radius-extra-large-default": {
+    "value": "{corner-radius-800}",
+    "type": "borderRadius",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "corner-radius-extra-large-default",
+        "constant-token-duplicate": false
+      }
+    }
+  },
+  "corner-radius-full": {
+    "value": "{corner-radius-1000}",
+    "type": "borderRadius",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "corner-radius-full",
+        "constant-token-duplicate": false
+      }
+    }
+  },
+  "corner-radius-small-size-small": {
+    "value": "{corner-radius-75}",
+    "type": "borderRadius",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "corner-radius-small-size-small",
+        "constant-token-duplicate": false
+      }
+    }
+  },
+  "corner-radius-small-size-medium": {
+    "value": "{corner-radius-small-default}",
```

Comment:
@nabuhasan this value should be corner-radius-100 (they should all point to the global tokens, not alias tokens)